excel_import_export/models/xlsx_template.py

```diff
@@ -208,10 +208,9 @@ def _update_result_field_common_wizard(self):
             _field.ensure_one()
             _field.write({"relation": self.result_model_id.model})
         _field.compute = """
-self['%s'] = self.env['%s'].search(self.safe_domain(self.domain))
-        """ % (
-            self.result_field,
-            self.result_model_id.model,
+self['{}'] = self.env['{}'].search(self.safe_domain(self.domain))
+        """.format(
```

Comment:
This is an unrelated module.

Reply:
@luisg123v  this change was made by the pre-commit stage. Without this change the pre-commit does not pass to build.